users/lamad/index.md

```diff
@@ -6,12 +6,33 @@ author: Adam Lam
 Adam Lam
 ================================
 
-lamad@oregonstate.edu
+<img src="{{ site.url }}users/harterj/images/jrh-sit.jpg" height="375" width="600">
 
-## Background
+Email: <a href="mailto:lamad@oregonstate.edu" target="top"> lamad@oregonstate.edu </a>
 
-I am a currently a first year Masters student in Nuclear Engineering at OSU, working with Dr. Todd Palmer as an advisor and project leader. I got my undergraduate degree in Physics and Minor in Mathematics at California State University, Sacramento. I am a member of Zigma Pi Zigma, an honors society for Physics.
 
 
-## Research
-  Currently I am looking into Levermore-Pomraning Binary Stochastic Mixing for Transport in 3D as a comparison with Benchmark results.
+### Background
+I am currently a third year graduate student in Nuclear Engineering at OSU, working with Dr. Todd Palmer as an advisor and project leader. I got my undergraduate degree in Physics and Minor in Mathematics at California State University, Sacramento (CSUS). During my time at CSUS, I was a member of Zigma Pi Zigma society in Physics. I joined the team in the Fall of 2016 and have been working in the field of transport and reactor physics since.
+
+I have worked on two different project (stochastic mxing of materials, and multiband cross section generations) during my first two years with Dr. Palmer. I've also had the opportunity to interned at Lawrence Livermore National Laboratory for two summers (2017 and 2018).
+
+### Current projects
+* Master's Thesis
+  * I am in the process of finishing my thesis and looking to defend in May/June 2019
+
+* Dissertation
+  * Currently doing literature review and collecting material to write my dissertation proposal
+
+
+### Publications
+
+1. Coline Larmier, Adam Lam, Patrick Brantley, Fausto Malvagi, Todd Palmer, Andrea Zoia, ''Monte Carlo chord length sampling for d-dimensional Markov binary mixtures,'' J Quant. Spectrosc. Radiat. Transfer, 204 (2018), pp. Pages 256-271
```

Comment:
Can you leave links to these publications? (if you want...)

Reply:
Yea, I'll look into the links. Thanks.